packages/analytics/metric-cards/src/components/MetricCardContainer.cy.ts

```diff
@@ -80,4 +81,66 @@ describe('<MetricCardContainer />', () => {
     cy.get(container).should('be.visible')
 
   })
+
+  // Should display up arrow icon, and green trend value text
+  it('Upward trend displayed for diff above threshold', () => {
+    cy.mount(MetricCardContainer, {
+      props: {
+        cards: [
+          {
+            currentValue: 0.3014,
```

Comment:
How is this test case working given that currently it's only looking at 3 decimals when determining the change?  Is it because of the `val * 100`?  I'm pretty unclear on the context for this...

Reply:
yes, that's correct. If we don't pass in `val * 100` to `formatChangeFn` this test would fail.